server/service/individualServices/AuthorizationApplication.js

```diff
@@ -280,4 +277,42 @@ async function ApplicationandReleaseNumber(applicationaName, ReleaseNumber, auth
         console.log(err);
     }    
     return isApplicationNameExit;
+}
+
+
+async function  getApplicationandReleaseNumberAtConfigFile(applicationName, releaseNumber) {
+    try{
+    let isApplicationNameExit = false
+    let isReleaseNumberExit = false
```

Comment:
typo in the variable declaration shall be corrected from isReleaseNumberExit to isReleaseNumberExist. Modify it in all the used references.